File: main/src/main/java/net/fwitz/math/main/binary/escape/MandelPiPlot.java
package net.fwitz.math.main.binary.escape;

public class MandelPiPlot extends MandelbrotPlot {
    public static void main(String[] args) {
        new MandelcubedPlot().render();
    }

    public MandelPiPlot() {
        super(Math.PI);
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/MandelPiPlot.kt
package net.fwitz.math.main.binary.escape

object MandelPiPlot : MandelbrotPlot(
    name = "MandelPi (Escape Time)",
    power = Math.PI
) {
    @JvmStatic
    fun main(args: Array<String>) = MandelPiPlot.render()
}